package.json

```diff
@@ -40,6 +40,7 @@
     "lucide-react": "^0.412.0",
     "next": "14.2.5",
     "next-pwa": "^5.6.0",
+    "next-share": "^0.27.0",
```

Comment:
whats this for  ???

Reply:
Package for next-share to make the social share button in the document page